source/_integrations/xiaomi_miio.markdown

```diff
@@ -1,37 +1,39 @@
 ---
-title: Xiaomi Miio
-description: Instructions how to integrate your Xiaomi Mi WiFi Repeater 2 within Home Assistant.
+title: "Xiaomi Miio"
+description: "Overvieuw of all Xiaomi Miio Integrations."
 ha_category:
-  - Presence Detection
+  - Hub
 ha_iot_class: Local Polling
-ha_release: 0.67
-ha_codeowners:
-  - '@rytilahti'
-  - '@syssi'
+ha_release: '0.110'
 ha_domain: xiaomi_miio
-ha_config_flow: true
 ---
 
-The `xiaomi_miio` device tracker platform is observing your Xiaomi Mi WiFi Repeater 2 and reporting all associated WiFi clients.
-
-Please follow the instructions on [Retrieving the Access Token](/integrations/vacuum.xiaomi_miio/#retrieving-the-access-token) to get the API token.
-
-To add a Xiaomi Mi WiFi Repeater device tracker to your installation, add the following to your `configuration.yaml` file:
-
-```yaml
-device_tracker:
-  - platform: xiaomi_miio
-    host: 192.168.130.73
-    token: YOUR_TOKEN
-```
-
-{% configuration %}
-host:
-  description: The IP address of your miio device.
-  required: true
-  type: string
-token:
-  description: The API token of your miio device.
-  required: true
-  type: string
-{% endconfiguration %}
+The `xiaomi_miio` integration supports many devices, this page gives an overvieuw of all platfoms with links to the documentation.
+
+## Xiaomi Gateway
+See https://www.home-assistant.io/integrations/gateway.xiaomi_miio/
+
+## Xiaomi device tracker (Xiaomi Mi WiFi Repeater 2)
+See https://www.home-assistant.io/integrations/device_tracker.xiaomi_miio/
+
+## Xiaomi Air Purifier
+see https://www.home-assistant.io/integrations/fan.xiaomi_miio/
+
+## Xiaomi Air Quality Index Monitor 
+see https://www.home-assistant.io/integrations/sensor.xiaomi_miio/
+
+see https://www.home-assistant.io/integrations/air_quality.xiaomi_miio/
+
+## Xiaomi IR Remote
+see https://www.home-assistant.io/integrations/remote.xiaomi_miio/
+
+## Xiaomi Mi Robot Vacuum
+see https://www.home-assistant.io/integrations/vacuum.xiaomi_miio/
+
+## Xiaomi Philips Light
+see https://www.home-assistant.io/integrations/light.xiaomi_miio/
+
+## Xiaomi Smart WiFi Socket and Smart Power Strip
+see https://www.home-assistant.io/integrations/switch.xiaomi_miio/
+
```

Comment:
Please consolidate the platform pages on the main integration page instead.

Reply:
done